src/PhaseState.jl

```diff
@@ -71,6 +72,29 @@ export getkf
     end
     return kfs
 end
+
+@inline function getkfs(ph::U,T::W1,P::W2,C::W3,ns::Q,V::W4,phi) where {U,W1<:ReverseDiff.TrackedReal,W2,Q<:AbstractArray,W3,W4<:Real}
```

Comment:
I had a fix at my local code using `kfs = similar(ns,length(ph.reactions))` to replace `kfs = zeros(Q.parameters[1],length(ph.reactions))`, do you think this would work?

Reply:
Nice! That's much simpler and it looks like it resolves the issue as well!